User Role Options (#126)
metricly/metricly-docs: content/account-admin/_index.md

Home to all of your [notifications][2], divided into the following categories: Email, OpsGenie, PagerDuty, SNS, Slack, and Webhook.

### Users
Create & assume users from the users panel. There are three types of users:

- **Administrator**:
  - Can create, update, delete everything but objects created by another user. All alerts can be deleted.

>   - Can create and assume Admin, Power, and Read-only users

Reviewer: ```suggestion
  - Can create and assume Administrator, Power User, and Read Only users.
```

  - Can create and assume sub-accounts
  - Cannot view the CloudWisdom Billing page
- **Power User**:
  - Can create, update, delete everything but objects created by another user. All alerts can be deleted.
  - Cannot create and assume users
  - Cannot create and assume sub-accounts
  - Cannot view the CloudWisdom Billing page

> - **Read Only User**:

Reviewer: ```suggestion
- **Read Only**:
```
Then approved!

Home to all of your [notifications][2], divided into the following categories: Email, OpsGenie, PagerDuty, SNS, Slack, and Webhook.

### Users

> Create & assume sub-accounts from the users panel. There are four types of users:

Reviewer: Sub-accounts can't be assumed from the users panel. That's the Sub-accounts page.

> - **Account Owners**:

Reviewer: Is this a role? I don't see it in the UI.

  - Can create and assume all user types
  - Can view the CloudWisdom Bill page

> - **Admins**:

Reviewer: Administrator is the role name.

  - Can create and assume Admin, Power, and Read-only users
  - Can create and assume sub-accounts
  - Cannot view the CloudWisdom Bill page

> - **Power Users**:

Reviewer: Power User is the role name.

Reviewer: ```suggestion
- **Power User**:
```

  - Cannot create and assume users
  - Cannot create and assume sub-accounts
  - Cannot view the CloudWisdom Bill page

> - **Read-only Users**:

Reviewer: Read Only is the role name.

  - Cannot create and assume users
  - Cannot create and assume and sub-accounts

>   - Cannot view the CloudWisdom Bill Page

Reviewer: Billing page.
```suggestion
  - Cannot view the CloudWisdom Billlling Page
```
